doc/articles/platform-specific-xaml.md

```diff
@@ -96,16 +96,40 @@ Consider the following XAML which is using the Windows Community Toolkit's [Blur
 
 The pre-defined prefixes are listed below:
 
-| Prefix        | Included platforms           | Excluded platforms           | Namespace                                                   | Put in `mc:Ignorable`? |
-|---------------|------------------------------|------------------------------|-------------------------------------------------------------|------------------------|
-| `win`         | Windows                      | Android, iOS, web, macOS     | `http://schemas.microsoft.com/winfx/2006/xaml/presentation` | no                     |
-| `xamarin`     | Android, iOS, web, macOS     | Windows                      | `http:/uno.ui/xamarin`                | yes                    |
-| `not_win`     | Android, iOS, web, macOS     | Windows                      | `http:/uno.ui/not_win`                | yes                    |
-| `android`     | Android                      | Windows, iOS, web, macOS     | `http:/uno.ui/android`                | yes                    |
-| `ios`         | iOS                          | Windows, Android, web, macOS | `http:/uno.ui/ios`                    | yes                    |
-| `wasm`        | web                          | Windows, Android, iOS, macOS | `http:/uno.ui/wasm`                   | yes                    |
-| `macos`       | macOS                        | Windows, Android, iOS, web   | `http:/uno.ui/macos`                  | yes                    |
-| `not_android` | Windows, iOS, web, macOS     | Android                      | `http://schemas.microsoft.com/winfx/2006/xaml/presentation` | no                     |
-| `not_ios`     | Windows, Android, web, macOS | iOS                          | `http://schemas.microsoft.com/winfx/2006/xaml/presentation` | no                     |
-| `not_wasm`    | Windows, Android, iOS, macOS | web                          | `http://schemas.microsoft.com/winfx/2006/xaml/presentation` | no                     |
-| `not_macos`   | Windows, Android, iOS, web   | macOS                        | `http://schemas.microsoft.com/winfx/2006/xaml/presentation` | no                     |
+| Prefix        | Included platforms                 | Excluded platforms                 | Namespace                                                   | Put in `mc:Ignorable`? |
+|---------------|------------------------------------|------------------------------------|-------------------------------------------------------------|------------------------|
+| `win`         | Windows                            | Android, iOS, web, macOS, Skia     | `http://schemas.microsoft.com/winfx/2006/xaml/presentation` | no                     |
+| `xamarin`     | Android, iOS, web, macOS           | Windows, Skia                      | `http:/uno.ui/xamarin`                                      | yes                    |
```

Comment:
Are you sure about this? The `xamarin` prefix includes Wasm, I would assume it also includes Skia. (Whether it *should* include them is another subject.)

Reply:
No, not sure about that and it was the main assumption I mentioned in other information of the PR. It doesnt make sense for Xamarin to include skia (wasm still uses mono), if that is how it is implemented I can change it though.

Reply:
Right, I didn't check the PR description properly.

I checked and the definition for `"xamarin"` hasn't changed, it's a blanket include that's synonymous with `"not_win"`:
https://github.com/unoplatform/uno/blob/master/src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props#L41

You're right that it doesn't make sense that it includes Skia, and really IMO it shouldn't include Wasm either (Mono != Xamarin). It'd be a significant breaking change to adjust it, though. (And worse, a silent breaking change in many cases.)

Reply:
Yea, you're right that this doesn't make sense but needs to stay for compatibility. I'll adjust so xamarin is the same as not_win.

Reply:
I realize skia has just been released a few months ago and isn't documented yet. While we can't really fix Wasm, it is possible to fix Skia in this case to exclude it from Xamarin. If that broke any existing code it would be a very, very small amount using technically undocumented assumptions. It's kind-of now or never to fix this.

Otherwise, I made the updates so `xamarin` = `non_win`

Reply:
The problem is that 'existing code' includes Uno code that was not specifically written for Skia (or even Wasm), where `xamarin` was used as a synonym for 'not UWP'. 

In the medium term it's just as likely that we move toward a different mechanism for conditional Xaml, in any case, because this approach is quite limited in certain use cases (eg when you want to prefix a type that's not coming from one of the default namespaces; when you want to include >1 and exclude >1 targets, eg iOS && macOS but not Android or WASM).